CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## 0.7.0
+
+- Restructure Configuration [#111]
```

Comment:
Should include details of how the config was changed and what the user needs to do. Should also not be added to the CHANGELOG until release but rather added as a code block in the body of the pull request (see: https://github.com/oracle/mysql-operator/pull/122)